cluster/install/fluo-cluster/bin/fluo-cluster

```diff
@@ -58,6 +61,7 @@ ready)
   echo "  start        Start all services on cluster"
   echo "  kill         Kill cluster processes"
   echo "  ready        Checks if cluster is running and ready"
+  echo "  ashell       Runs Accumulo shell"
```

Comment:
In addition to this or maybe instead of this, could create an alias in bashrc that runs the accumulo shell